### ISSUE TITLE
Fix handling for batch cancel endpoints

### ISSUE THREAD

**Issue by tiagosiebler** [tiagosiebler/binance#336]
const  orderIdList = [ 3118081111, 3118081110 ];
await this.client.cancelMultipleOrders({ symbol, orderIdList: JSON.stringify(orderIdList) as any })

https://t.me/nodetraders/23604

**Comment by movy** [comment]
Still experiencing this, for those googling:
```
/cancelMultipleOrders usdm {"symbol":"DASHUSDT","orderIdList":[581240933]}  response:
{
  code: -1130,
  msg: "Data sent for parameter 'orderidlist' is not valid."
}
```

**Comment by tiagosiebler** [comment]
Will be fixed shortly in the next release (v2.8.5), as soon as #344 is merged and the CI run finishes.

**Comment by tiagosiebler** [comment]
Released, let me know if you find any other issues. Thanks!
